nemoguardrails/colang/v2_x/runtime/eval.py

```diff
@@ -120,27 +128,44 @@ def replace_with_index(name, _match):
 
     # Finally, just evaluate the expression
     try:
+        # TODO: replace this with something even more restrictive.
+        functions: Dict[str, Callable] = {
+            "len": len,
+            "flow": system_functions.flow,
+            "action": system_functions.action,
+            "regex": _create_regex,
+            "search": _regex_search,
+            "find_all": _regex_findall,
+            "uid": new_uid,
+            "str": _to_str,
+            "simple_str": _simple_str,
+            "escape": _escape_string,
+            "is_int": _is_int,
+            "is_float": _is_float,
+            "is_bool": _is_bool,
+            "is_str": _is_str,
+            "is_regex": _is_regex,
+            "less_than": _less_than_operator,
+            "equal_less_than": _equal_or_less_than_operator,
+            "greater_than": _greater_than_operator,
+            "equal_greater_than": _equal_or_greater_than_operator,
+            "not_equal_to": _not_equal_to_operator,
```

Comment:
Are these still needed, now that we can use these directly in expressions?

Reply:
Yes, for now its for event matching based on inequalities for numbers:
`match Event(number = less_than(3))`
Ideally we could extend the parsing to support:
`match Event(number < 3)`